CHANGELOG.md

```diff
@@ -176,31 +62,31 @@ The following are deprecated and will be removed in a future release. For anythi
 - ImageryTileLayer.rasterInfo deprecated since 4.29. Use serviceRasterInfo instead.
 - LayerList.multipleSelectionEnabled deprecated since 4.29. Use selectionMode instead.
 - LayerList.selectionEnabled deprecated since 4.29. Use selectionMode and dragEnabled instead.
-- Lighting.clone deprecated since version 4.24
-- Lighting deprecated since version 4.24. Use SunLighting instead.
+- ListItemPanel.className deprecated since version 4.30. Use icon
+- ListItemPanel.className deprecated since version 4.30. Use icon
```

Comment:
This is a duplicate (I realize there is also a duplicate in the release notes -- but they are linking to different classes.. LayerList vs TableList)

Reply:
Done.